packages/app/src/settings/SettingsCard.vue

```diff
@@ -43,20 +45,87 @@
 
 <script lang="ts" setup>
 import type { FunctionalComponent, SVGAttributes } from 'vue'
+import { ref, ComponentPublicInstance, computed, watchEffect, nextTick } from 'vue'
 import Collapsible from '@cy/components/Collapsible.vue'
 import ListRowHeader from '@cy/components/ListRowHeader.vue'
+import { useRoute } from 'vue-router'
 
-defineProps<{
+const props = defineProps<{
   title: string
+  name?: string
```

Comment:
a bit nit picky but `name` is a bit generic of a prop here. `section` or `sectionName` would be a bit more descriptive IMO